email_alert_service/models/email_alert.rb

```diff
@@ -43,4 +44,24 @@ def email_api_client
   def strip_empty_arrays(tag_hash)
     tag_hash.reject { |_, tags| tags.empty? }
   end
+
+  def document_links
+    strip_empty_arrays(
+      document.fetch("links", {}).merge("taxon_tree" => taxon_tree)
+    )
+  end
+
+  def taxon_tree
+    return [] unless document.dig("links", "taxons")
+
+    # TODO: update the public API of the taxonomy helpers gem to also accept a
+    # single fully expanded document so that we don't have to pass in the
+    # document twice.
+    linked_content_item = GovukTaxonomyHelpers::LinkedContentItem.from_publishing_api(
+      content_item: document,
+      expanded_links: document,
```

Comment:
@MatMoore double checked the payload the publishing API puts on the queue and it looks like `expanded_links` is present https://github.com/alphagov/publishing-api/blob/master/app/presenters/edition_presenter.rb#L59.